bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8_madspin_card.dat

```diff
@@ -0,0 +1,26 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#*    Manual:                                               *
+#*    cp3.irmp.ucl.ac.be/projects/madgraph/wiki/MadSpin     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+# set seed 1
+# set Nevents_for_max_weight 75 # number of events for the estimate of the max. weight
+# set BW_cut 15                 # cut on how far the particle can be off-shell
+# set spinmode onshell          # Use one of the madspin special mode
```

Comment:
When just uploading an example, please clean these lines up. They just complicate the review.